applications/plugins/Geomagic/src/Geomagic/GeomagicDriver.cpp

```diff
@@ -346,6 +346,7 @@ void GeomagicDriver::initDevice()
     updatePosition();
     sofa::core::objectmodel::BaseObject::d_componentState.setValue(sofa::core::objectmodel::ComponentState::Valid);
 #else
+    msg_error() << "GeomagicDriver initialisation failed because OpenHaptics (HD/hd.h) has not been found. Please install openHaptics and specify OpenHaptics_DIR at CMake configure stage.";
```

Comment:
```suggestion
    msg_error() << "GeomagicDriver initialization failed because the Geomagic plugin was built without OpenHaptics (HD/hd.h). If you build the plugin yourself, install OpenHaptics and specify OpenHaptics_DIR at CMake configure stage.";
```

As this message is intended for users, I would rephrase like that

Reply:
It was to compile the plugin on the CI, where openHaptics can't be installed,  to catch any error due to change in SOFA base of code.
This was the solution 3 years ago. Maybe WE can install HD now on the CI?

Reply:
Ok for the change I let you validate the final version during the meeting.